apps/meeting/server.js

```diff
@@ -2,189 +2,104 @@
 // SPDX-License-Identifier: MIT-0
 
 /* eslint-disable */
+const express = require('express');
 const compression = require('compression');
-const fs = require('fs');
-const url = require('url');
 const { v4: uuidv4 } = require('uuid');
-const AWS = require('aws-sdk');
-/* eslint-enable */
-
-let hostname = '127.0.0.1';
-let port = 8080;
-let protocol = 'http';
-let options = {};
-
-const chime = new AWS.Chime({ region: 'us-east-1' });
-const alternateEndpoint = process.env.ENDPOINT;
-
-// Optional features like Echo Reduction is only available on Regional Meetings API
-// https://docs.aws.amazon.com/chime/latest/APIReference/API_Operations_Amazon_Chime_SDK_Meetings.html
-const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
-const chimeRegionalEndpoint = process.env.REGIONAL_ENDPOINT || 'https://meetings-chime.us-east-1.amazonaws.com';
```

Comment:
Just to clarify, we remove the ability to specify alternate endpoint here so in other words, we would not be able to test gamma endpoint or other region endpoint.
I think its probably fine as we usually just use JS SDK demo for that but just let you know.

Reply:
I checked our deploy and release script, we were not using `REGIONAL_ENDPOINT`, so I just removed it. Let's do those test in JS SDK demo.